### DISCUSSION TITLE
How can I show Devtools in production?

### DISCUSSION THREAD

**Discussion by DerJacques** [TanStack/query#2444] [Q&A]
According to the docs and my experience, React Query Devtools are automatically hidden in production environments.
Before the Devtool was merged into the main repo, the docs contained a note saying that this could be circumvented by importing the Devtool directly via `react-query/dist/react-query-devtools.production.min.js`.

Unfortunately, this doesn't work with the latest version of React Query and TypeScript, since no module definition exists for this import.

Do you know if there is any other way to enable the Devtools in production?

**Comment by TkDodo** [comment]
There’s this twitter thread: https://twitter.com/tannerlinsley/status/1392857647037091849

also a suggestion to make this somewhat official on the docs. If you like the solution or find a better one, I’d appreciate a PR to the docs
